scripts/obj.js

```diff
@@ -0,0 +1,22 @@
+function runFunction() {
```

Comment:
All your functions in these files are in global scope (you can check it in console, they are all available). Since that the naming of these function should be specific.`runFunction` is too common.

Reply:
Ok)